app/src/main/java/org/p2p/wallet/moonpay/ui/new/PaymentMethodViewHolder.kt

```diff
@@ -0,0 +1,33 @@
+package org.p2p.wallet.moonpay.ui.new
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import org.p2p.wallet.R
+import org.p2p.wallet.databinding.ItemPaymentMethodBinding
+import org.p2p.wallet.moonpay.model.PaymentMethod
+import org.p2p.wallet.utils.viewbinding.getString
+
+class PaymentMethodViewHolder(
+    private val binding: ItemPaymentMethodBinding,
+    private val onClickListener: (PaymentMethod) -> Unit
+) : RecyclerView.ViewHolder(binding.root) {
+
+    constructor(
+        parent: ViewGroup,
+        onClickListener: (PaymentMethod) -> Unit
+    ) : this(
+        ItemPaymentMethodBinding.inflate(LayoutInflater.from(parent.context), parent, false),
```

Comment:
we can create ViewHolders much easier - org.p2p.wallet.home.ui.main.empty.PopularTokenViewHolder